aspnetcore/grpc/json-transcoding-openapi/Program.cs

```diff
@@ -0,0 +1,20 @@
+#region snippet_1
```

Comment:
You can drop the snippet tags when they cover the whole file. You do **not** need to change any highlight line numbers in the text for this change.

```suggestion
```